.drone.star

```diff
@@ -727,7 +727,11 @@ def main(ctx):
     return pipelines + deploys + checkStarlark()
 
 def beforePipelines(ctx):
-    return yarnlint() + checkForRecentBuilds(ctx) + changelog(ctx) + website(ctx) + cacheOcisPipeline(ctx)
+    title = ctx.build.title.lower()
+    if "[unit-test-only]" in title or "[unit-tests-only]" in title:
```

Comment:
```suggestion
    if "unit-test-only" in title or "unit-tests-only" in title:
```
We don't usually require the `[]` - that is just a habit that we are in when writing PR titles.

Reply:
Yeah, I was making it strict. As my PR title has `unit-tests-only` in description part `[unit-tests] Run minimum pipelines for unit-tests-only PRs`, this is also treated as unit-test-only and runs minimum pipelines.

Reply:
It's tricky to know what to do exactly - if someone puts `(unit-tests-only)` then that should also be OK.